frontend/public/style/_overrides.scss

```diff
@@ -214,6 +214,11 @@ tags-input .autocomplete .suggestion-item em {
   }
 }
 
+// Until Patternfly-React-Extensions is updated: https://github.com/patternfly/patternfly-react/issues/1146
+.catalog-tile-pf-title {
+  word-break: break-word;
```

Comment:
```suggestion
  @include co-break-word;
```

Reply:
Fixed. Moved to _catalog.scss so the mixin is available.